packages/react/src/components/accordion/accordion-styles.ts

```diff
@@ -0,0 +1,96 @@
+import styled, { css } from 'styled-components';
+import { Theme } from '../../themes';
+
+type StyledFunction = (isExpanded: boolean, theme: Theme) => ReturnType<typeof css>;
+type AccordionSectionFunction = (theme: Theme) => ReturnType<typeof css>;
+
+export const StyledAccordionGroup = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+`;
+
+export const AccordionSectionStyled: AccordionSectionFunction = (theme) => css`
+    background: ${theme.greys['neutral-02']};
+    border: 0 solid ${theme.greys['neutral-15']};
+    border-radius: 0 0 var(--border-radius-2x) var(--border-radius-2x);
+    border-top-width: 0;
+    margin-bottom: var(--spacing-1x);
+    max-height: 0;
+    overflow: hidden;
+    transition: max-height 0.5s ease, border 0.5s ease;
+    &.expanded {
+        border: 1px solid ${theme.greys['neutral-15']};
+        border-top-width: 0;
+        max-height: 500px;
```

Comment:
This could be problematic if the content has more than 500px.

Reply:
I can make it 99999px as it doesnt affect and I doubt anyone will have anything with that size right?
but I need to implement it due to height is not useful for the animation.
If you have any suggestion here I would appreciate.